js/src/interface.js

```diff
@@ -283,12 +324,8 @@ let app = new Vue({
 
 // Fired when the external save button is clicked
 Fliplet.Widget.onSaveRequest(() => {
-
   // Send back the result
-  Fliplet.Widget.save(JSON.parse(JSON.stringify({
-    settings: settings,
-    result: app.result
-  }))).then(() => {
+  Fliplet.Widget.save(app.result).then(() => {
```

Comment:
@squallstar I didn't know that's what `JSON parse+stringify` does, so I can probably put that back in. Just searched and couldn't find a better way to do this :-/ so let me revert this.

I'm only returning results because, well, I'm not sure why the settings are returned. It's not used when using `dataSource.fetchWithOptions()`, and the provider callee has the `settings` information anyway.